src/browser/modules/D3Visualization/components/ResultsPane.tsx

```diff
@@ -137,49 +146,63 @@ export class LegendComponent extends Component<any, State> {
         return (
           <StyledLegendInlineListItem key={i} data-testid="viz-legend-reltypes">
             <StyledLegendContents className="contents">
-              <StyledTokenRelationshipType
-                onClick={onClick}
-                style={style}
-                className="token token-relationship-type"
+              <Popup
+                on="click"
+                basic
+                pinned
+                trigger={
+                  <StyledTokenRelationshipType
+                    onClick={onClick}
+                    style={style}
+                    className="token token-relationship-type"
+                  >
+                    {legendItemKey}
+                    <StyledTokenCount className="count">
+                      {`(${numberToUSLocale(
+                        legendItems[legendItemKey].count
+                      )})`}
+                    </StyledTokenCount>
+                  </StyledTokenRelationshipType>
+                }
+                wide
+                inverted={this.props.theme === DARK_THEME}
```

Comment:
added `inverted` as otherwise in dark theme, there was a light padding around the grass editor. There may be another way around it